test/command/video-audio.md

```diff
@@ -0,0 +1,19 @@
+```
+% pandoc -f markdown-implicit_figures -t html
+![](./test.mp4)
+
+![Your browser does not support video.](foo/test.webm){width=300}
```

Comment:
Seems like you have not updated the test file

Reply:
No, that's the alt text... see a few lines above.